### PR TITLE
docs: Improve README quality

### PR DESCRIPTION
**Pull request by elishaestibeiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,17 @@ If you do not already have these installed, a manager like [nvm](https://github.
 
 Download the latest package from this repo.
 
-Install light-server using ```npm install -g light-server```
+Install light-server using:
```
Comment (alvinphilips) [inline README.md:14]: You can remove "using" here

File: README.md [hunk 2]
```diff
@@ -11,9 +11,17 @@ If you do not already have these installed, a manager like [nvm](https://github.
...
 
-Install light-server using ```npm install -g light-server```
+Install light-server using: 
 
-In the root directory of the downloaded folder, run ```light-server --serve .```
+```
+npm install -g light-server
+```
+
+In the root directory of the downloaded folder, run
```
Comment (alvinphilips) [inline README.md]: Add a colon at the end :)

File: README.md [hunk 3]
```diff
@@ -37,20 +45,34 @@ In the root directory of the downloaded folder, run ```light-server --serve .```
 ### Install Rust and wasm-pack
 Install Rust using [rustup](https://www.rust-lang.org/tools/install)
 
-Check that Rust installed successfully using ```rustc --version```
+Check that Rust installed successfully using:
```
Comment (alvinphilips) [inline README.md]: "using" can be removed here as well

File: README.md [hunk 4]
```diff
@@ -37,20 +45,34 @@ In the root directory of the downloaded folder, run ```light-server --serve .```
...
+
+```
+rustc --version
+```
 
 Install [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/).
 
 ### Ensuring npm is up-to-date
-Make sure your version of npm is up-to-date with ```npm install npm@latest -g```
+Make sure your version of npm is up-to-date with:
```
Comment (alvinphilips) [inline README.md]: remove "with"

File: README.md [hunk 5]
```diff
@@ -37,20 +45,34 @@ In the root directory of the downloaded folder, run ```light-server --serve .```
...
+
+```
+ npm install npm@latest -g
```
Comment (alvinphilips) [inline README.md]: remove space before npm

File: README.md [hunk 6]
```diff
@@ -78,9 +104,18 @@ cd www
 npm install
 ```
 
-To run a live development server, use the command ```npm run start```
+To run a live development server, use the command:
```
Comment (alvinphilips) [inline README.md]: remove "the command"

File: README.md [hunk 7]
```diff
@@ -78,9 +104,18 @@ cd www
...
+npm run start
+```
+
+To build the application, use:
 
-To build the application, use ```npm run build```. The files will be stored in a folder called dist/ within the www directory.
+```
+npm run build
+``` 
+The files will be stored in a folder called dist/ within the www directory.
```
Comment (alvinphilips) [inline README.md]: Add an empty line before this

File: README.md [hunk 8]
```diff
@@ -78,9 +104,18 @@ cd www
...
```
Comment (alvinphilips) [inline README.md:120]: Remove a newline here

**Review by alvinphilips — CHANGES_REQUESTED** [review]
I've requested a few changes, everything else looks good :) Good job!

File: README.md (hunk 2 above)
Comment (elishaestibeiro) [reply README.md]: Done! Sorry about that :)

**Pull request merged**